nx-X11/lib/X11/Imakefile

```diff
@@ -211,9 +211,11 @@ XCURSOR_DEFINES = -DUSE_DYNAMIC_XCURSOR -DLIBXCURSOR=\"libXcursor.so.$(SOXCURSOR
            SRCS = $(SRCS1) $(SRCS2) $(SRCS3) $(EXTRASRCS) /* try *.c if you have trouble */
        LINTLIBS =
 
-EXTRASTATICSRC = $(STATICLCSRCS)
+EXTRASTATICSRCS = $(STATICLCSRCS)
+EXTRASRCS = $(EXTRASTATICSRCS)
```

Comment:
Good point. Amended. (I have the impression, that I have a more recent version of this branch on my notebook, but anyway, fixing it now,).